src/sqs/queues.ts

```diff
@@ -12,18 +12,33 @@ const LONG_POLLING_INTERVAL_SEC = 3;
 const logger = getLogger("sqs-queues");
 
 // TODO: Make this a class
-export const sqsQueues = {
-	backfill: new SqsQueue<BackfillMessagePayload>({
+
+let backfillQueue: SqsQueue<BackfillMessagePayload> | undefined = undefined;
+
+const backfillQueueMessageSender = (message, delaySec, logger) => {
```

Comment:
This block seems "complicated, can we do following instead?

`await Promise.race([this.messageHandler(context, this <-- pass itself in and `this` is guarantee to be initialised!), timeoutPromise]);`

Reply:
So in the `sync/installation.ts` just do `this.sendMessage` instead of `sqs.backfill.sendMessage`